api/serviceregistry/serviceregistry.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+
+package svreg;
+option go_package = "serviceregistry";
+
+
+service Registry{
+  rpc RegisterWorkload (ServiceWorkload) returns (Empty) {} // Request register of a new service
+  rpc QueryService (ServiceRequest) returns (ServiceWorkloads) {} // Request a services
```

Comment:
Comment should be updated to something similar to "Request workloads for a particular service"

Reply:
done